backend/Ballerina-Runner/src/main/java/org/wso2/msf4j/ballerina/Application.java

```diff
@@ -52,7 +52,7 @@
 
     // Extract the Ballerina distribution and copy psfinder script.
     static {
-        String DISTRIBUTION_ZIP = "ballerina-0.87.zip";
+        String DISTRIBUTION_ZIP = "ballerina-0.92.zip";
```

Comment:
Is this scalable? Isn't there a better way to do this?

Reply:
+1, we need to find way to parameterize this version. Eg:- Maven profiling.